data.sql

```diff
@@ -1,7 +1,16 @@
 /* Populate database with sample data. */
 
```

Comment:
![image](https://github.com/anyars-encarta/vet-clinic/assets/93039359/b81cc9fe-afbe-4487-87e5-278ce8bfd736)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you fix the License link in your readme file to make your project licensed and your readme descriptive and professional 😊
- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍